schema/project-level/project-schema.json

```diff
@@ -460,6 +460,16 @@
           "uniqueItems": true,
           "minItems": 1
         },
+        "environmentalMeasures": {
```

Comment:
As with the codelist, move this and climateMeasures under conservationMeasures (and order them alphabetically).